lib/kafkaClient.js

```diff
@@ -695,6 +726,29 @@ KafkaClient.prototype.clearCallbackQueue = function (socket, error) {
   delete this.cbqueue[socketId];
 };
 
+// Sends a version 1 metadata request which includes controller node in response
+KafkaClient.prototype.loadMetadataForTopicsV2 = function (topics, cb) {
+  var correlationId = this.nextId();
+  var request = protocol.encodeMetadataV1Request(this.clientId, correlationId, topics);
+  var broker = this.brokerForLeader();
+
+  this.queueCallback(broker.socket, correlationId, [protocol.decodeMetadataV1Response, cb]);
+  broker.write(request);
+};
+
+KafkaClient.prototype.createTopicsV2 = function (topics, cb) {
```

Comment:
I'm inclined to just call this `createTopics`.  Planning to drop support node 4 so we're making major version change and this can be included in the things that "break".